### PR TITLE
Update repo standards

### PR DESCRIPTION
**Pull request by timbussmann**
the dotsettings file seems to be changing quite a few values. Not sure whether SC was following the guidelines so far, so that file might have to be reverted? @Particular/servicecontrol-maintainers

### CODE REVIEW

**Comment by timbussmann** [comment]
seems like SC heavily violates the coding guidelines defined in the repo standards settings. I'll remove the dotsettings changes as there are over 800 inspection errors. I'll have a closer look and create a followup issue or PR.

File: src/nuget.config [hunk 1]
```diff
@@ -1,5 +1,10 @@
 <?xml version="1.0" encoding="utf-8"?>
 <configuration>
+  <packageRestore>
```
Comment (bording) [inline src/nuget.config]: This is definitely not needed here, and I don't think we really need it anywhere, as mentioned on the RepoStandards PR.

Comment (timbussmann) [reply src/nuget.config]: I'm fine with removing it

**Comment by weralabaj** [comment]
@timbussmann are you going to continue or can I take over this PR to address https://github.com/Particular/ServiceControl/issues/1250?

**Comment by timbussmann** [comment]
@weralabaj I think this PR should not be used to address #1250 as that will introduce many changes. Let's merge this one and create a new PR for #1250.

**Comment by weralabaj** [comment]
👍

**Comment by danielmarbach** [comment]
Happy to merge this and mute the inspections until #1250 is done. Thoughts?

**Comment by WilliamBZA** [comment]
:+1:

**Pull request merged**